### PR TITLE
Merge fix startgame lobby

### PR DESCRIPTION
**Pull request by JerroldJV**
Please review - I will be doing some additional development today based off of this branch

### CODE REVIEW

**Comment by JerroldJV** [comment]
FYI- this is stuff that had already been reviewed in previous requests.  Once somebody does a quick sanity check we should merge.

File: clueless/models.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ class Space(models.Model):
     """
     posX = models.IntegerField()
     posY = models.IntegerField()
-    spaceNorth = models.OneToOneField('self', related_name='spaceSouth', blank=True)
-    spaceEast = models.OneToOneField('self', related_name='spaceWest', blank=True)
+    spaceNorth = models.OneToOneField('self', related_name='spaceSouth', blank=True, null=True)
+    spaceEast = models.OneToOneField('self', related_name='spaceWest', blank=True, null=True)
     spaceCollector = models.ForeignKey(SpaceCollection)
```
Comment (rsfriedman) [inline clueless/models.py:38]: So I had to look up what null = true does but according to django documentation doing this is potentially bad and unnecessary if we have blank=true also? I might be misunderstanding why you added this but I don't know if we need it

https://docs.djangoproject.com/en/1.10/ref/models/fields/

File: clueless/settings.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['0.0.0.0']
```
Comment (rsfriedman) [inline clueless/settings.py:25]: can we change this?

File: clueless/views.py [hunk 3]
```diff
@@ -1,16 +1,97 @@
...
+			else:
+				return redirect('index')
+		else: #user didn't login right, give them a login error
+			loginError = True
+
+	#either the authentication failed, or they are coming to the login page for the first time.  Render the page
+	template = loader.get_template('clueless/login.html')
+	context = Context({'loginError':loginError})
+	return HttpResponse(template.render(context, request))
+
```
Comment (rsfriedman) [inline clueless/views.py:53]: so when i do this and don't have a username yet i get the following:
i feel like there should be a much better way of displaying an error...

ProgrammingError at /login/
relation "auth_user" does not exist
LINE 1: ...user"."is_active", "auth_user"."date_joined" FROM "auth_user...
                                                             ^
Request Method:	POST
Request URL:	http://0.0.0.0:8000/login/
Django Version:	1.10.2
Exception Type:	ProgrammingError
Exception Value:	
relation "auth_user" does not exist
LINE 1: ...user"."is_active", "auth_user"."date_joined" FROM "auth_user...
                                                             ^
Exception Location:	/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py in execute, line 64
Python Executable:	/usr/local/bin/python
Python Version:	3.6.0
Python Path:	
['/code',
 '/usr/local/lib/python36.zip',
 '/usr/local/lib/python3.6',
 '/usr/local/lib/python3.6/lib-dynload',
 '/usr/local/lib/python3.6/site-packages']
Server time:	Wed, 30 Nov 2016 01:35:46 +0000

File: clueless/views.py [hunk 4]
```diff
@@ -60,3 +141,194 @@ def playerturn(request):
...
+			player = Player()
+			player.user = user
+			player.character = character
+
+			# Constructs our game, saves the changes and starts it
+			game = Game()
+			game.addPlayer(player)
+			game.save() # TODO: Check if this is to do be done before startGame()
+			game.startGame()
+	else:
```
Comment (rsfriedman) [inline clueless/views.py:184]: So I thought we decided logic wise that there would be options to start game or create game, but this method seems to automatically just start a game??

File: clueless/views.py [hunk 5]
```diff
@@ -60,3 +141,194 @@ def playerturn(request):
...
+			whoWhatWhere = WhoWhatWhere()
+			whoWhatWhere.character = character
+			whoWhatWhere.weapon = weapon
+			whoWhatWhere.room = room
+
+
+
+
+def make_suggestion(request):
+	"""
```
Comment (rsfriedman) [inline clueless/views.py:220]: why are there duplicate methods?

File: clueless/views.py [hunk 6]
```diff
@@ -60,3 +141,194 @@ def playerturn(request):
...
+			whoWhatWhere.weapon = weapon
+			whoWhatWhere.room = room
+
+			accusation = Accusation()
+			accusation.whoWhatWhere = whoWhatWhere
+	else:
+		logger.error('POST expected, actual ' + request.method)
+
+def make_accusation(request):
+	"""
```
Comment (rsfriedman) [inline clueless/views.py:311]: again a duplicate method

**Comment by rsfriedman** [comment]
made a bunch of comments--overall it seems like it will be ok but since i can't login yet i can't view the actual page....

File: clueless/models.py (hunk 1 above)
Comment (JerroldJV) [reply clueless/models.py:38]: I found out it was needed on Postgres for blank FK references (or at least it wasn't working when I tried).

File: clueless/settings.py (hunk 2 above)
Comment (JerroldJV) [reply clueless/settings.py:25]: @GregHilston changed this, it doesn't seem to make any difference to me.

File: clueless/views.py (hunk 3 above)
Comment (JerroldJV) [reply clueless/views.py:53]: Weird?  I'm not sure why you get this issue.

File: clueless/views.py (hunk 4 above)
Comment (JerroldJV) [reply clueless/views.py:184]: Updated in latest pull request- see that

File: clueless/views.py (hunk 5 above)
Comment (JerroldJV) [reply clueless/views.py:220]: Huh....

File: clueless/settings.py (hunk 2 above)
Comment (rsfriedman) [reply clueless/settings.py:25]: yeah no actual issue with it just seems like a weird ip to use

File: clueless/views.py (hunk 6 above)
Comment (JerroldJV) [reply clueless/views.py:311]: Probably f'd up a merge conflict.  Will fix in latest branch.

File: clueless/views.py (hunk 3 above)
Comment (rsfriedman) [reply clueless/views.py:53]: maybe my lack of database default set ups? i'll see if i still get it after fixing the python issues

**Pull request closed without merging**